.github/workflows/reusable_build_and_test_wheels.yml

```diff
@@ -189,7 +189,6 @@ jobs:
           manylinux: manylinux_2_31
           container: off
           command: build
-          sccache: "true"
```

Comment:
Will maturin still *use* sccache if this is set to false though?

Reply:
Yes. All this does is install sccache and setup the env variables. During the build, it calls out to `cargo` which calls `rustc` which calls `sccache`.

Reply:
https://github.com/PyO3/maturin-action/blob/v1/src/index.ts#L557